lib/resty/jwt.lua

```diff
@@ -464,43 +502,84 @@ function _M.load_jwt(self, jwt_str, secret)
   return jwt_obj
 end
 
---@function validate_exp_nbf - validate expiry and not valid before
---@param jwt_obj
-local function validate_exp_nbf(jwt_obj, is_validation_loose, leeway)
-  if jwt_obj[str_const.reason] ~= nil then
-    return
-  end
-
+--@function validate exp nbf claims - validate expiry and not valid before
+--@param jwt_obj, leeway, require_nbf_claim, require_exp_claim
+local function validate_lifetime(jwt_obj, leeway, require_nbf_claim, require_exp_claim)
   local exp = jwt_obj[str_const.payload][str_const.exp]
   local nbf = jwt_obj[str_const.payload][str_const.nbf]
 
+  if (leeway ~= nil and exp == nil and nbf == nil) then
```

Comment:
by default, leeway(`lifetime_grace_period` is 0),
so, a jwt with without lifetime claims is an invalid jwt?

maybe we should set default `lifetime_grace_period` to `nil`?